variables.tf

```diff
@@ -57,3 +57,15 @@ variable "slack_message_text" {
   description = "The content of the message sent to Slack directly"
   default     = ""
 }
+
+variable "deployment_bucket" {
+  type        = string
+  description = "The bucket that hold the lambda deployment asset"
+  default     = "TODO"
```

Comment:
If you leave out the default then folks are forced to include it. Would that be a better approach here?

Reply:
Be gentle here, just a draft ;)

The idea is we would set this to a default to point to Truss S3 bucket. Unfortunately the deployment asset has to be stored in the same region as the lambda. Have an ADR for that https://github.com/trussworks/legendary-waddle/pull/210.

I think in the end I'll pull out the default and provide a list of s3 paths. Not 100% for sure how to make the user experience simple. If you have ideas I'm all 👂 !